### DISCUSSION TITLE
11.1.0-rc2

### DISCUSSION THREAD

**Discussion by maxkatz6** [AvaloniaUI/Avalonia#16213] [General]
**Full Changelog**: https://github.com/AvaloniaUI/Avalonia/compare/11.1.0-rc1...11.1.0-rc2

## What's Changed
* Convert IAvaloniaXamlIlParentStackProvider to eager one if needed by @MrJul in https://github.com/AvaloniaUI/Avalonia/pull/16028
* Fix negation of null values. by @grokys in https://github.com/AvaloniaUI/Avalonia/pull/16101
* Fix BindingNotification handling in MultiBinding  by @grokys in https://github.com/AvaloniaUI/Avalonia/pull/16102
* Fix `ICommand` bindings in style setters by @grokys in https://github.com/AvaloniaUI/Avalonia/pull/16122
* Make bindings react to PropertyChanged even if property hasn't changed by @grokys in https://github.com/AvaloniaUI/Avalonia/pull/16150

<hr /><em>This discussion was created from the release <a href='https://github.com/AvaloniaUI/Avalonia/releases/tag/11.1.0-rc2'>11.1.0-rc2</a>.</em>

**Comment by koustubhmoharir** [comment]
Is #16120 planned for 11.1? A crash when typing into a text box is a showstopper. Currently I have switched to a font that does not have ligatures on fi or fl, but I am not sure if there are other sequences that may have ligatures.

**Reply by stevemonaco** [reply]
It's to my understanding that only serious regressions will be fixed for 11.1.0 in any potential future RC. Since that ligature bug existed on 11.0.x and was fixed recently, it should be backported to a future 11.1.x version along with other fixes that were PR'd.

**Reply by maxkatz6** [reply]
It will go to 11.1.1.

**Comment by npolyak** [comment]
inching closer!
